src/common/ui/code.vue

```diff
@@ -486,6 +494,27 @@ $selectionDarkBg: rgba(73, 72, 62, .99);
   }
 }
 
+/* CodeMirror show-hints fix to work here */
+.CodeMirror-hints {
+  z-index: 9999;
+  /* consistent font with the rest, copied from .monospace-font rule from style/style.css */
+  font-family: "Fira Code", Menlo, Monaco, Consolas, "Liberation Mono"
+                , "Courier New", Courier, monospace;
```

Comment:
Instead of copying simply add `.CodeMirror-hints,` to the original rule.

Reply:
It we add `.CodeMirror-hints` to the original rule in `style/style.css`, it will be overriden by bundled `codemirror/addon/hint/show-hint.css` with `font-family: monospace;`

We could add `!important` to the original rule, but I'm uncertain whether it might has other unintended consequence. Thoughts?

Reply:
Let's simply use a specificity hack e.g. `.CodeMirror .CodeMirror-hints`

Reply:
I used the `.CodeMirror-hints.eclipse` instead, because hints UI elements are directly under `<body>`, outside `.CodeMirror`, or violentmonkey UI specific `.editor-code`

Other alternatives inlcude `body > .CodeMirror-hints` , `ul.CodeMirror-hints`, but they depend on `shot-hints` addon internals in what and where HTMLs are generated. Adding `.eclipse` has the advantage that it is the style violentmonkey explicitly uses.

HTMLs for the generated hints
```html
   ...
   <ul class="CodeMirror-hints eclipse" style="left: 62.4px; top: 438.4px;">...
</body>
```